aws/sentry/aurora-postgres.tf

```diff
@@ -0,0 +1,204 @@
+  default     = "postgres"
+}
+
+# Don't use `admin` 
+# Read more: <https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html>
+# ("MasterUsername admin cannot be used as it is a reserved word used by the engine")
+variable "postgres_admin_user" {
+  type        = string
+  description = "Postgres admin user name"
+  default     = "sentry"
+}
+
+# Must be longer than 8 chars
+# Read more: <https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html>
+# ("The parameter MasterUserPassword is not a valid password because it is shorter than 8 characters")
+variable "postgres_admin_password" {
+  type        = string
+  description = "Postgres password for the admin user"
+  default     = ""
+}
+
+variable "postgres_db_name" {
+  type        = string
+  description = "Postgres database name"
+  default     = "sentry"
+}
+
+variable "aurora_postgres_engine_version" {
+  type        = string
+  description = "Database Engine Version for Aurora PostgeSQL"
+  default     = "9.6.12"
+}
+
+variable "aurora_postgres_cluster_family" {
+  type        = string
+  description = "Database Engine Version for Aurora PostgeSQL"
+  default     = "9.6.12"
+}
+
+# db.r4.large is the smallest instance type supported by Aurora Postgres 9.6
+# Postgres 10.6 and later can use db.r5.large
+# Postgres 10.7 and later can use db.t3.medium
+# https://aws.amazon.com/rds/aurora/pricing
+variable "postgres_instance_type" {
+  type        = string
+  default     = "db.r4.large"
+  description = "EC2 instance type for Postgres cluster"
+}
+
+variable "postgres_cluster_size" {
+  type        = number
+  default     = "2"
+  description = "Postgres cluster size"
+}
+variable "postgres_autoscaling_enabled" {
+  type        = bool
+  default     = false
+  description = "Set true to enable the database cluster autoscaler"
+}
+variable "postgres_autoscaling_min_capacity" {
+  type        = number
+  default     = 1
+  description = "Minimum number of Postgres instances to be maintained by the autoscaler"
+}
+variable "postgres_autoscaling_max_capacity" {
+  type        = number
+  default     = 6
+  description = "Maximum number of Postgres instances to be maintained by the autoscaler"
+}
+
+variable "postgres_cluster_enabled" {
+  type        = bool
+  default     = "true"
+  description = "Set to false to prevent the module from creating any resources"
+}
+
+variable "postgres_iam_database_authentication_enabled" {
+  type        = string
+  default     = "false"
+  description = "Specifies whether or mappings of AWS Identity and Access Management (IAM) accounts to database accounts is enabled."
+}
+
+resource "random_string" "postgres_admin_password" {
+  count   = local.postgres_cluster_enabled ? 1 : 0
+  length  = 26
+  special = true
+}
+
+locals {
+  # Some of these locals used to be computed. Keep them as local just for consistency,
+  # even though now they are just copies of variables.
+  postgres_cluster_enabled = var.postgres_cluster_enabled
```

Comment:
don't need this local in TF 0.12 (var. postgres_cluster_enabled should be a bool)

Reply:
I know, but like I said in the comment, all the postgres vars are locals this way, which is a nice consistency.